Emby.Naming/Common/NamingOptions.cs

```diff
@@ -328,6 +328,10 @@ public NamingOptions()
 
                 // *** End Kodi Standard Naming
 
+                // [bar] Foo - 1 [baz]
+                new EpisodeExpression(@".*?(\[.*?\])+.*?(?<seriesname>(\w+\s)+?)[-\s_]+(?<epnumber>\d{1,3}).*$"){
```

Comment:
any reason to limit episode numbers to 999?

Reply:
Other expressions also limit the episode number to 3 digits. I wanted to keep this consistent but it is an easy change to increase it.

Reply:
I would suggest removing the limit on number of digits where it is appropriate in a regexp altogether, I see no real reason for limiting the amount of episodes in a single season.